tests/gradients/test_parameter_shift_cv.py

```diff
@@ -0,0 +1,1105 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the gradients.parameter_shift_cv module."""
+import pytest
+
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane.gradients import param_shift_cv
+from pennylane.gradients.parameter_shift_cv import (
+    _grad_method,
+    _gradient_analysis,
+    _transform_observable,
+)
+
+
+hbar = 2
+
+
+class TestGradAnalysis:

```

Comment:
This is a copy and paste of existing tests: https://github.com/PennyLaneAI/pennylane/blob/ab71001bc55c2945a0945f42131c45d77da3512e/tests/tape/test_cv_param_shift.py#L25